### PR TITLE
import numpy as np

### PR DESCRIPTION
**Pull request by agramfort**

### CODE REVIEW

File: pyriemann/utils/covariance.py [hunk 1]
```diff
@@ -61,12 +63,12 @@ def covariances(X, estimator='cov'):
     estimator : {'cov', 'scm', 'lwf', 'oas', 'mcd', 'corr'} (default: 'scm')
         covariance matrix estimator:
 
-        * 'cov' for numpy based covariance matrix, https://numpy.org/doc/stable/reference/generated/numpy.cov.html
+        * 'cov' for numpy based covariance matrix, https://np.org/doc/stable/reference/generated/np.cov.html
```
Comment (qbarthelemy) [inline pyriemann/utils/covariance.py:66]: URL must not be corrected.

File: pyriemann/utils/covariance.py [hunk 2]
```diff
@@ -61,12 +63,12 @@ def covariances(X, estimator='cov'):
     estimator : {'cov', 'scm', 'lwf', 'oas', 'mcd', 'corr'} (default: 'scm')
         covariance matrix estimator:
 
-        * 'cov' for numpy based covariance matrix, https://numpy.org/doc/stable/reference/generated/numpy.cov.html
+        * 'cov' for numpy based covariance matrix, https://numpy.org/doc/stable/reference/generated/np.cov.html
```
Comment (qbarthelemy) [inline pyriemann/utils/covariance.py:66]:
```suggestion
        * 'cov' for numpy based covariance matrix, https://numpy.org/doc/stable/reference/generated/numpy.cov.html
```

File: pyriemann/utils/covariance.py [hunk 3]
```diff
@@ -61,12 +63,12 @@ def covariances(X, estimator='cov'):
...
         * 'scm' for sample covariance matrix, https://scikit-learn.org/stable/modules/generated/sklearn.covariance.empirical_covariance.html
         * 'lwf' for shrunk Ledoit-Wolf covariance matrix, https://scikit-learn.org/stable/modules/generated/sklearn.covariance.ledoit_wolf.html
         * 'oas' for oracle approximating shrunk covariance matrix, https://scikit-learn.org/stable/modules/generated/sklearn.covariance.OAS.html
         * 'mcd' for minimum covariance determinant matrix, https://scikit-learn.org/stable/modules/generated/sklearn.covariance.MinCovDet.html
-        * 'corr' for correlation coefficient matrix, https://numpy.org/doc/stable/reference/generated/numpy.corrcoef.html
+        * 'corr' for correlation coefficient matrix, https://numpy.org/doc/stable/reference/generated/np.corrcoef.html
```
Comment (qbarthelemy) [inline pyriemann/utils/covariance.py:71]:
```suggestion
        * 'corr' for correlation coefficient matrix, https://numpy.org/doc/stable/reference/generated/numpy.corrcoef.html
```

**Comment by agramfort** [comment]
good to go from my end @qbarthelemy

**Review by qbarthelemy — APPROVED** [review]

**Comment by kingjr** [comment]
aaaaah finally 
img

**Comment by agramfort** [comment]
thx @qbarthelemy for the quick review !

**Pull request merged**